walk-in-interview/src/main/java/com/google/job/data/JobsDatabase.java

```diff
@@ -42,6 +43,17 @@
                 .set(job);
     }
 
+    /**
+     * Updates a specific field.
+     *
+     * @param jobId Cloud Firestore Id of the job post.
+     */
+    public void updateJobId(String jobId) {
+        DocumentReference documentReference = FireStoreUtils.getFireStore().collection(JOB_COLLECTION).document(jobId);
+        // (async) Update one field
```

Comment:
Don't quite understand this. You query a document matching a job id? then update its job id field to the same job id?

Reply:
Yep, since when the job post is added into the database, it still has no `jobId` (the `jobId` is empty string). Once it is added, the cloud firestore will automatically generate a id for this job post, then update this auto-generated id into that job post `jobId` field, which will be convenient for later retrieving